22.md

```diff
@@ -0,0 +1,111 @@
+NIP-22
+======
+
+Comment
+-------
+
+`draft` `optional`
+
+A comment is meant to be used to comment on
+or reply to anything except event kinds
+whose specifications have already defined other
+ways to comment on or reply to them.
+
+For example, it can be used to comment on a [NIP-23](23.md) `kind:30023` nostr blog post
+or on an off-nostr news article or to reply to another comment.
+
+## Structure
+
+A comment is an event of `kind:1111` with plaintext `.content`
+(no HTML, no Markdown nor other formatting).
+
+**It must have a `k` tag** pointing to the kind of the subject being commented on.
+Its value has a prefix: `"n:"` for nostr subjects and other prefixes (listed in a section ahead)
```

Comment:
why prefix with `n`? Seems like it would be better to just use ints for kinds like we do elsewhere, and use prefixes for other stuff.